Building.md

```diff
@@ -0,0 +1,49 @@
+This is the building guide for FullScreenShenanigans projects.
+
+### Table of Contents
+
+1. [Obtaining the Source Code](#obtaining-the-source-code)
+2. [Build Process](#build-process)
+3. [Using the Console](#using-the-console)
+
+## Obtaining the Source Code
+
+In order to play a FullScreenShenanigans game, you first need to obtain the source code. There are two ways to do this:
+
+A. Clone the project repository using GitHub:
+1. If you don't already have Git installed on your computer, go to [this page](http://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and follow the instructions to download Git.
+2. Once Git is downloaded, follow [these directions](https://help.github.com/articles/cloning-a-repository/) to clone the repository.
+
+B. Download the latest release .zip of the project, extract that onto your computer, and open index.html  in a browser (preferably Google Chrome).
+
+
+## Build Process
+
+FullScreenShenanigans projects use [Grunt](http://gruntjs.com/) to automate building, which requires [Node.js](http://node.js.org). 
+
+Go to the root repository folder and run the following commands:
+
+    npm install
+    grunt
+    
+You may also have to manually install grunt first:
+
+    npm install -g grunt grunt-cli
+
+Every time grunt is run, it carries out a set of tasks requested in the Gruntfile found in the repository. In the case of the FullScreenShenanigan projects, grunt is used to create a new Distribution folder in the root. It also converts .ts files into .js files, compresses the project's main .js file, and creates sourcemaps to improve debugging capabilities. It's also zipped for your convenience.
+
+You can now play the game by opening index.html in a browser (preferably Google Chrome).
+
+
+## Using the Console
+You can use the console to customize your gaming experience. Open the console by pressing `Ctrl+Shift+J` at the same time. A few examples of console use when playing FullScreenMario are pausing the game by typing
```

Comment:
For code samples in a specific, language, use:

```markdown
```language-name
...
```
```

...where language-name is `javascript` or `typescript`.